label_sync/kubeflow_label.yml

```diff
@@ -3,6 +3,8 @@ labels:
   name: addition/feature
 - color: d2b48c
   name: area/api
+- color: d2b48c
+  name: area/0.4.0
```

Comment:
Can you add 
previously:
  - name: release/0.4.0

As in the sections below.

This way all issues currently tagged release/0.4.0 will be retagged 0.4.0